pipelines/geo-equipamientos/dev_run.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Documentation 
+#
+# https://drive.google.com/file/d/1U23HS_EQQwv8C-mcC-0rjAQiFFA_kJI2/view
+
+set -e
+source .env
```

Comment:
This env file is not required as  far as I can see, the variables you are defining on it are not used.